resources/js/Pages/Admin/Rooms/UpdateRoomForm.vue

```diff
@@ -1,98 +1,195 @@
+                        <option value="unavailable">Unavailable</option>
+                    </select>
+                    <jet-input-error :message="form.error('status')" class="mt-2"/>
+                </div>
+
+                <div class="m-6">  
+                    <jet-label for="equippment" value="Equippment"/>
+                    <br>
+                    <label class="flex items-center">
+                    <input type="checkbox" class="form-checkbox" v-model="form.projector"/>
+                    <span class="ml-2 text-sm text-black">Projector</span>    
+                    </label> 
+                    <label class="flex items-center">
+                    <input type="checkbox" class="form-checkbox" v-model="form.television"/>
+                    <span class="ml-2 text-sm text-black">Television</span>  
+                    </label> 
+                    <label class="flex items-center">
+                    <input type="checkbox" class="form-checkbox" v-model="form.computer"/>
+                    <span class="ml-2 text-sm text-black">Computer</span>
+                    </label>   
+                    <label class="flex items-center"> 
+                    <input type="checkbox" class="form-checkbox" v-model="form.whiteboard"/>
+                    <span class="ml-2 text-sm text-black">Whiteboard</span>    
+                    </label>                 
+                    <jet-input-error :message="form.error('equippment')" class="mt-2"/>
+                </div>
+
+                <div class="m-6">  
+                    <jet-label for="restrictions" value="Restrictions"/>
+                    <br>
+                    <label class="flex items-center">
+                    <input type="checkbox" class="form-checkbox" v-model="form.ambiant_music"/>
+                    <span class="ml-2 text-sm text-black">Ambiance Music</span>    
+                    </label> 
+                    <input type="checkbox" class="form-checkbox" v-model="form.sale_for_profit"/>
+                    <span class="ml-2 text-sm text-black">Sale for Profit</span>  
+                    </label> 
+                    <label class="flex items-center">
+                    <input type="checkbox" class="form-checkbox" v-model="form.fundraiser"/>
+                    <span class="ml-2 text-sm text-black">Fundraiser</span>
+                    </label>    
+                    <jet-input-error :message="form.error('restrictions')" class="mt-2"/>
+                </div>
+
+                <div class="m-6">  
+                    <jet-label for="furniture" value="Furniture"/> 
+                    <br>                        
+                    <label class="flex items-center text-sm">Sofa(s)</label>
+                    <jet-input  v-model.number="form.sofas" type="number"/> 
+
+                    <label class="flex items-center">Coffee Table(s)</label>
+                    <jet-input  v-model.number="form.coffee_tables" type="number"/>
+
```

Comment:
![image](https://user-images.githubusercontent.com/21128579/105226501-0d053380-5b2e-11eb-99a0-4b06f6407d3e.png)
Is there a way to add some 'min' restriction on all necessary fields for front end validation to not send negative numbers?

Reply:
There's  a min property, I've added it.. it's flawless.